.github/workflows/codespell.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       - name: Codespell
         uses: codespell-project/actions-codespell@v2
         with:
-          ignore_words_list: Crate,crate
+          ignore_words_list: Crate,crate,elease
```

Comment:
intentional?

Reply:
The regex is `[Rr]elease`. Yet, perhaps the whole idea of having a shell script is ludicrous. See the comment below. By pythonizing, that exception will be gone.